lib/index.js

```diff
@@ -9,38 +9,35 @@ const internals = {
     }
 };
 
-
-exports.register = (server, options, next) => {
-
-    // Clone defaults to prevent override
-    const defaults = Object.assign({}, internals.defaults);
-    const response = Object.assign(defaults, options.response);
-
-    const config = {
-        auth: !!options.auth,
-        handler: (request, reply) => reply.response(response),
-        response: {
-            schema: Joi.object({
-                ver: Joi.string().regex(/[0-9]+(\.[0-9]+)*/).default('0.0.0'),
-                sha: Joi.string().alphanum().length(7).default('plzSet1')
-            }),
-            modify: true,
-            options: {
-                allowUnknown: true,
-                stripUnknown: false,
-                convert: true
+module.exports = {
+    pkg: require('../package.json'),
+    register: (server, options) => {
+
+        // Clone defaults to prevent override
+        const defaults = Object.assign({}, internals.defaults);
+        const response = Object.assign(defaults, options.response);
+
+        const config = {
+            auth: !!options.auth,
+            handler: (request, reply) => reply.response(response),
+            response: {
+                schema: Joi.object({
+                    ver: Joi.string().regex(/[0-9]+(\.[0-9]+)*/).default('0.0.0'),
+                    sha: Joi.string().alphanum().length(7).default('plzSet1')
```

Comment:
can we use `'undefined'` (string) instead of `plzSet1`?

Reply:
it can be `'undefnd'` because it have a 7 size limit